src/data/calendarData.js

```diff
@@ -5,4 +5,50 @@ export default [
     'startDate': new Date(2017, 6, 24),
     'endDate': new Date(2017, 6, 24)
   }
```

Comment:
Did you get a chance to test to start the app and make sure the code you added worked? This code does't compile because of the missing `,` at the end of the line. Once the app starts, I'd check to make sure the events are showing up on the calendar.